app/controllers/admin/admin_invitations_controller.rb

```diff
@@ -43,9 +43,9 @@ def find
     end
     if !invitation_params[:token].blank?
       @invitation = Invitation.find_by(token: invitation_params[:token])
-    elsif !invitation_params[:invitee_email].blank?
-      @invitations = Invitation.where('invitee_email LIKE ?', "%#{invitation_params[:invitee_email]}%")
-      @invitation = @invitations.first if @invitations.length == 1
+    elsif invitation_params[:invitee_email].present?
+      @invitations = Invitation.where("invitee_email LIKE ?", "%#{invitation_params[:invitee_email]}%").presence
+      @invitation = @invitations&.first
```

Comment:
This works, but is there a reason we can't check `unless @user || @invitation.present? || @invitations.present?` on line 51? It might be a little clearer about when the error gets displayed, imo.